### ISSUE TITLE
alignments get worse when continue training

### ISSUE THREAD

**Issue by EuphoriaCelestial** [NVIDIA/tacotron2#381]
the alignment of my new tacotron model suddenly get worse after checkpoint_368000. As you can see in the picture below (checkpoint_384000), after it synthesize the input sentence, it added a very long "random voice and echo" 
![ảnh](https://user-images.githubusercontent.com/43977734/85650151-141aaf00-b6cf-11ea-84cb-fa50a3046154.png)

this is the audio of the synthesize, the sentence only last until 00:11:
[checkpoint384.zip](https://github.com/NVIDIA/tacotron2/files/4828967/checkpoint384.zip)

all previous checkpoints is okay, this is example of checkpoint_368000
![ảnh](https://user-images.githubusercontent.com/43977734/85650396-a8851180-b6cf-11ea-8e41-1c3af3903533.png)

is it getting overfit or it just getting over the local minimum? should I stop the training or continue?

**Comment by rafaelvalle** [comment]
You should resume training if training and validation loss are both going down.

**Comment by EuphoriaCelestial** [comment]
> You should resume training if training and validation loss are both going down.

no these numbers not go down anymore; but my experience tell me that it still can be improved, especially in noise reduction.
my previous model has training and validation loss stuck around 0,3 but the audio still getting better and less noise in output audio, as I mentioned here https://github.com/NVIDIA/tacotron2/issues/353

my biggest concern is this area, why is it appeared in my synthesize?
![ảnh](https://user-images.githubusercontent.com/43977734/85812946-2ad30b80-b78c-11ea-9e93-daafa743fdf1.png)

**Comment by rafaelvalle** [comment]
That has happened because your gate did not activate when it was supposed to.
Try adjusting the gate threshold.

**Comment by EuphoriaCelestial** [comment]
> That has happened because your gate did not activate when it was supposed to.
> Try adjusting the gate threshold.

can I change it after the model finished training? or I need to adjust it and re-train?
and what exactly do I need to do? increase or decrease the threshold? I see it's 0.5 by default

**Comment by rafaelvalle** [comment]
You can adjust after the model is trained. Decreasing the threshold will close the gate earlier.

**Comment by EuphoriaCelestial** [comment]
> You can adjust after the model is trained. Decreasing the threshold will close the gate earlier.

many thank, I have successfully fixed by setting threshold to 0.1 - 0.2
but will it cause any side effect?

**Comment by rafaelvalle** [comment]
Take a look at other phrases.

**Comment by EuphoriaCelestial** [comment]
> Take a look at other phrases.

after testing for a week, I found no side effect. Thank you for your support! 
Closed.

**Comment by anderleich** [comment]
Which is the effect of the gate threshold on inference?
